translations/templates/password-recovery/es-ar.yaml

```
@@ -13,3 +13,6 @@ back: Volver
 passwordChanged: Contraseña cambiada
 helloAgain: ¡Hola de nuevo!
 passwordHasBeenChanged: 'Tu contraseña ha sido cambiada. '
+expiredTokenTitle: El enlace ha expirado.
+expiredTokenMessage: 'Puedes solicitar otro enlace haciendo '
+expiredTokenLink: click aqui
```

En toda la app usamos español rioplatense, yo lo mantendría por coherencia

```suggestion
expiredTokenMessage: 'Podés solicitar otro enlace haciendo '
expiredTokenLink: click acá
```